pancancer_utilities/data_utilities.py

```diff
@@ -0,0 +1,99 @@
+import os
+
+import pandas as pd
+import pickle as pkl
+from sklearn.model_selection import KFold
+from sklearn.preprocessing import MinMaxScaler
+
+import pancancer_utilities.config as cfg
+
+def load_expression_data(subset_mad_genes=cfg.num_features_raw,
+                         scale_input=False,
+                         verbose=False):
+    """Load and preprocess saved TCGA gene expression data.
+
+    Arguments
+    ---------
+    subset_mad_genes (int): TODO still need to implement this
+    scale_input (bool): whether or not to scale the expression data
+    verbose (bool): whether or not to print verbose output
+
+    Returns
+    -------
+    rnaseq_df: samples x genes expression dataframe
+    """
+    if verbose:
+        print('Loading gene expression data...')
+
+    rnaseq_df = pd.read_csv(cfg.rnaseq_data, index_col=0, sep='\t')
```

Comment:
Just double checking, the file is stored in samples x genes format , not genes x samples?

Reply:
Yep, the processed RNA-seq data (being loaded into memory here) is samples x genes. The original file from the Genome Data Commons is genes x samples, but in my data download/preprocessing script `00_download_data.ipynb` I transpose it (among other things).

For me, it's easier to store it this way since I can load it and put it straight into scikit-learn classifiers. I'm planning to subset by both genes and samples, so I don't think either orientation will be substantially better in terms of runtime (but I'll keep this in mind going forward).

Reply:
That's totally fair! I hadn't seen any samples x genes files before so wanted to make sure